Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Repository/ProjectionContentGraph.php

```diff
@@ -262,7 +262,15 @@ public function determineHierarchyRelationPosition(
                 ]
             )->fetchAssociative();
 
+            if (!$succeedingSiblingRelation) {
+                throw new \Exception(
+                    'Could not fetch succeeding sibling relation',
```

Comment:
The error message could contain some more details (affected nodes, dsp, ...)

Reply:
I adjusted the error message and added the available information